cass-operator.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: cass-operator
+  version: 1.18.2
+  epoch: 0
+  description: Manages Cassandra cluster as standalone product or as part of the k8ssandra-operator
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - build-base
+      - go
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/k8ssandra/cass-operator.git
+      tag: v${{package.version}}
+      expected-commit: f1f25eb8434876f4eb519a2e5504c171020916ad
+
+  - runs: go get golang.org/x/net@v0.17.0 # Mitigate CVE-2023-39325 and CVE-2023-3978
+
+  - runs: CGO_ENABLED=0 go build -a -o manager cmd/main.go
+
+  - runs: mkdir -p ${{targets.destdir}}/usr/bin; install -Dm755 ./manager ${{targets.destdir}}/usr/bin/manager
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: k8ssandra/cass-operator
+    strip-prefix: v
+    use-tag: true
```

Comment:
Looks like this repo publishes releases:
- https://github.com/k8ssandra/cass-operator/releases

We generally will favour releases over tags, assuming the repo publishes releases. 

```suggestion
```